src/main/java/carpet/CarpetServer.java

```diff
@@ -1,5 +1,6 @@
 package carpet;
 
+import java.io.IOException;
```

Comment:
automated import? seems unused

Reply:
Not automated, seems like I forgot it after running the code to force load all classes, removed now.